packages/sdk/src/CLRunner.ts

```diff
@@ -41,6 +41,34 @@ export enum SessionStartFlags {
 export interface InternalCallback<T> extends CLM.Callback, ICallback<T> {
 }
 
+export const convertInternalCallbackToCallback = <T>(c: InternalCallback<T>): CLM.Callback => {
+    const { logic, render, mockResults, ...callback } = c
+
+    const resultsWithoutUndefinedValues = mockResults.map(r => {
```

Comment:
nit : `resultsWithDefinedValues` would be easier to understand than the double-negative

Reply:
I agree on double negatives being confusing but in this cause it more precisely describes what the code does.  I would not consider `null` a defined value but yet it is included.

Changed to `resultsWithUndefinedAsNull` to hopefully remove negative and be more clear.